src/steamship/agents/tools/question_answering/vector_search_learner_tool.py

```diff
@@ -15,7 +15,7 @@ class VectorSearchLearnerTool(VectorSearchTool):
     name: str = "VectorSearchLearnerTool"
     human_description: str = "Learns a new fact and puts it in the Vector Database."
     agent_description: str = (
-        "Used to remember a fact. Only use this tool if someone asks to remember or learn something. ",
+        "Used to remember a fact. Only use this tool if someone asks to remember or learn something.",
```

Comment:
I think I wrote this original description with a bug.

In Python:

("S1", "S2", "S3") is a tuple of strings, but
("S1" "S2" "S3") is a concatenation of strings.

So:

* The commas in this list probably need to get removed
* The space after something. should be added back

Then the result will be the single string that was intended.

Alternative would be we use a different method of string concat that's less error prone -- my bad!

Reply:
Should be fixed now.